_unittest_solvers/test_26_emit.py

```diff
@@ -30,7 +30,8 @@ def aedtapp(add_app):
 
 
 @pytest.mark.skipif(is_linux, reason="Emit API fails on linux.")
-@pytest.mark.skipif(sys.version_info < (3,8) or sys.version_info >=(3, 12), reason="Emit API is only available for Python 3.8+.")
+@pytest.mark.skipif(sys.version_info < (3,10), reason="Emit API is only available for Python 3.10+.")
```

Comment:
```suggestion
@pytest.mark.skipif(sys.version_info < (3, 8), reason="Emit API is only available for Python 3.8+.")
```
For the time being we need to let people test `pyaedt` with Emit on python 3.8 (e.g. they use 2024.2). Once we get rid of the compatibility with python 3.8 in `pyaedt` (planned with `v1.0.0`), I suggest that we apply your changes.

Reply:
I applied a similar change here, separating the logic for before/after 2025R1.